packages/typespec-test/test/openai_generic/generated/typespec-ts/src/api/audio/translations/index.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CreateTranslationRequest,
+  CreateTranslationResponse,
+} from "../../../models/models.js";
+import {
+  CreateTranslation200Response,
+  CreateTranslationDefaultResponse,
+  isUnexpected,
+  OpenAIContext as Client,
+} from "../../../rest/index.js";
+import {
+  StreamableMethod,
+  operationOptionsToRequestParameters,
+} from "@azure-rest/core-client";
+import { CreateTranslationOptions } from "../../../models/options.js";
+
+export function _createTranslationSend(
+  context: Client,
+  audio: CreateTranslationRequest,
+  options: CreateTranslationOptions = { requestOptions: {} }
+): StreamableMethod<
+  CreateTranslation200Response | CreateTranslationDefaultResponse
+> {
+  return context
+    .path("/audio/translations")
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      contentType: (options.contentType as any) ?? "multipart/form-data",
+      body: {
+        file: audio["file"],
+        model: audio["model"],
+        prompt: audio["prompt"],
+        response_format: audio["responseFormat"],
+        temperature: audio["temperature"],
+      },
+    });
+}
+
+export async function _createTranslationDeserialize(
+  result: CreateTranslation200Response | CreateTranslationDefaultResponse
+): Promise<CreateTranslationResponse> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    text: result.body["text"],
+  };
+}
+
+export async function createTranslation(
```

Comment:
I noticed that in the https://github.com/joheredi/unbranded_lib/blob/main/src/api/audio/translations/index.ts#L51 manual code, the function here is called `create`, I wonder if this is from previous typespec or we should generate something to replace the duplicated part into a empty string if it's the same as the interface name ?

Reply:
We should use the name in TypeSpec. The manual code I had was assuming typespec called `create`. Operations don't have to be globally unique, just unique in the namespace.

Reply:
I'm fine appending something to the interface name to avoid duplicates, such as operation hierarchy